### PR TITLE
Cindy text

### PR DESCRIPTION
**Pull request by cindylee0113**
Cleaned up the tags when user decides to go on a rampage and click the button 100000 times :D. So if a user has [b]Hi[/b] and selects to unbold Hi, the tags go away. Also changed the hyperlinks so that they open in a new tab :3 

(Also cleaned up the repetitive codes in the javascript file)

### CODE REVIEW

**Comment by cindylee0113** [comment]
Also just made it required to type at least a letter into message and conversation name!

File: src/main/webapp/javascript/textChange.js [hunk 1]
```diff
@@ -501,122 +425,83 @@ function heartEmoji(){
...
+		if (document.getElementById(id).style.borderStyle == "inset"){
 			var string = document.getElementsByName('message')[0].value;
 			var start = document.getElementsByName('message')[0].selectionStart;
 			var end = document.getElementsByName('message')[0].selectionEnd;
-			document.getElementsByName('message')[0].value = 
-			string.substring(0, start) + "[/b]" + string.substring(start, end) + 
-			"[b]" + string.substring(end);
+			if (string.substring(start - 3, start) == startTag && string.substring(end, end + 4) == endTag){
+				document.getElementsByName('message')[0].value = 
+				string.substring(0, start - 3) + string.substring(start, end) + string.substring(end + 4);
```
Comment (aljonpineda) [inline src/main/webapp/javascript/textChange.js]: Does this handle the chaining of tags that you mentioned?

Comment (cindylee0113) [reply src/main/webapp/javascript/textChange.js]: Yup! The function checks if there's tags before and after it . If there are and its the correct tags (like start _____ end or end ______ start) , then it gets rid of the tags.

Comment (aljonpineda) [reply src/main/webapp/javascript/textChange.js]: Ok! Looks pretty good!

**Pull request merged**